pkg/ccl/changefeedccl/changefeed_stmt.go

```diff
@@ -498,7 +504,7 @@ func createChangefeedJobRecord(
 	opts changefeedbase.StatementOptions,
 	jobID jobspb.JobID,
 	telemetryPath string,
-) (*jobs.Record, error) {
+) (*jobs.Record, uint, error) {
```

Comment:
Hmm I don't love having to add this as a return value given it's not directly related to creating the changefeed job record.

Reply:
Maybe you can have this function return `targets` instead and pass that around?